convert2rhel/toolopts/__init__.py

```diff
@@ -117,6 +118,17 @@ def set_opts(self, key, value):
         if value and not current_attribute_value:
             setattr(self, key, value)
 
+    def update_opts(self, key, value):
+        """Update a given option in toolopts.
+
+        :param key:
+        :type key: str
+        :param value:
+        :type value: str
+        """
+        if key and value:
+            setattr(self, key, value)
+
```

Comment:
Doesn't seem to be used anywhere

Reply:
Ah, yes! This is for the other PR I have. Forgot to remove from here.